.vscode/launch.template.json

```diff
@@ -0,0 +1,21 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+    {
+      "type": "node",
+      "protocol": "inspector",
+      "request": "launch",
+      "name": "Mocha Tests (currently opened test)",
+      "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
```

Comment:
I’m not clear on why this config is helpful.

Reply:
This is the setting to run the debugger with vscode.